libs/vkd3d/vkd3d_private.h

```diff
@@ -2471,15 +2470,13 @@ struct vkd3d_copy_image_pipeline_key
     const struct vkd3d_format *format;
     VkImageViewType view_type;
     VkSampleCountFlagBits sample_count;
-    VkImageLayout layout;
```

Comment:
Could there be packing shenanigans here? (memcmp checks for hashmap)?

Reply:
Not a new problem but I suppose we should memset the key first.

Reply:
Fixed.